docs/source/installation/configuration.md

```diff
@@ -225,9 +226,13 @@ security:
 
 ### Keycloak
 
-The `security.keycloak` section allows you to specify an initial password for the `root` user (to login at `https://myqhubsite.com/auth/admin/`) to manage your Keycloak database, for example add users/groups.
+The `security.keycloak` section allows you to specify an initial password for the `root` user (to login at `https://myqhubsite.com/auth/admin/`) to manage your Keycloak database.
+
+> NOTE: After the initial deployment, changing this password will have no affect.
+
+We strongly recommend changing this `initial_root_password` after your initial deployment and deleting this value from your `qhub-config.yaml`.
```

Comment:
I know what you mean here, but those two paragraphs are negating each other.

Reply:
I see what you mean. I have updated the language 👍